src/skottie-player.ts

```diff
@@ -17,16 +64,111 @@ export class SkottiePlayer extends LitElement {
     }
   `;
   }
+  
+  /**
+   * The title of the json file. This is displayed above the animation.
+   */
+  @property({type: String})
+  title = '';
+
+  /**
+   * The content of the json file.
+   */
+  @property({type: Object})
+  content:lottieFile = {};
+
+  /**
+   * The URL/path to the json file.
+   */
+  @property({type: String})
+  fileURL = '../samples/animations/gear.json';
+
+  /**
+   * The width of the animation.
+   */
+  @property({type: Number})
+  width = 0;
+
+  /**
+   * The height of the animation.
+   */
+  @property({type: Number})
+  height = 0;
 
   constructor() {
     super();
+    this.title = 'Select a file';
   }
 
   render(){
     return html`
       <h1>${this.title}</h1>
+
+      <div style='width:${this.width}px;height:${this.height}px' id='player' class='lottie' data-anim-loop='true'></div>
```

Comment:
try to keep lines under 80 characters long. you can move the different attributes to a new line.